docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -518,6 +520,162 @@ In order to see the current maintenance state on a node:
 
   # /usr/local/pf/bin/cluster/maintenance
 
+'''
+
+Maintenance and Operation
+--------------------------
+
+Shutting down a PacketFence Active/Active cluster of tree nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to shutdown the cluster in a proper way, here is the steps you need to follow.
+We are here assuming that we are on a cluster of tree nodes, on Centos7.
+
+We will shutdown the nodes, one by one, starting by one of the "Slaves" and end with the "Master".
+
+Find the Master Server: ::
+* Type *ip a* on each nodes, and look for the one which hold the VIP (the /32 address).
+
+WARNING: *YOU TAKE THE RISK OF LOOSING SOME DATA IF YOU DO NOT IDENTIFY MASTER AND SLAVES NODES* 
```

Comment:
I hate when my data becomes loose

But anyway, that statement isn't accurate, you shouldn't lose any data whether or not you follow a proper shutdown order